apps/subs/src/features/subs/domain/entities/subs.table.ts

```diff
@@ -0,0 +1,16 @@
+import { Column, Table, Model, DataType } from 'sequelize-typescript';
+
+@Table
+export class UserSubscription extends Model<UserSubscription> {
+  @Column({ type: DataType.STRING })
+  followerId: string;
+
+  @Column({ type: DataType.STRING })
+  followingId: string;
+
+  @Column({
+    type: DataType.DATE,
+    defaultValue: DataType.NOW,
+  })
+  createdAt: Date;
+}
```

Comment:
```suggestion
export class UserSubscription extends Model<UserSubscription> {
  @Column({ type: DataType.STRING })
  followerId: string;

  @Column({ type: DataType.STRING })
  followingId: string;

  @Column({
    type: DataType.DATE,
    defaultValue: DataType.NOW,
  })
  createdAt: Date;
  static createSubscription(followerId: string, followingId: string): UserSubscription {
    const subscription = new UserSubscription();
    subscription.followerId = followerId;
    subscription.followingId = followingId
    return subscription;
  }
}
```
It’s essential to adhere to DDD principles when creating entities. I recommend implementing a factory method for UserSubscription that encapsulates the responsibility of creation. This approach not only hides the complexities associated with instantiating the subscription but also promotes better separation of concerns. By doing so, you ensure that any business logic related to the creation of a subscription is managed within the factory, making the code more maintainable and easier to test.